packages/eslint-plugin/src/rules/import-playwright-a11y.cjs

```diff
@@ -32,8 +37,8 @@ module.exports = {
       if (isDisallowedImport) {
         context.report({
           node,
-          loc: node.loc,
-          message: `Import "${node.imported.name}" from "../../playwright-axe" instead because Onyx uses custom Playwright fixtures for providing a global configuration for accessability testing.`,
+          loc: node.loc ?? undefined,
+          message: `Import "${node.imported.name}" from "../../playwright/a11y" instead because Onyx uses custom Playwright fixtures for providing a global configuration for accessability testing.`,
```

Comment:
```suggestion
          message: `Import "${node.imported.name}" from "../../playwright/a11y" instead because onyx uses custom Playwright fixtures for providing a global configuration for accessability testing.`,
```